horace_core/sqw/coord_transform/@spher_proj/spher_proj.m

```diff
@@ -76,6 +78,7 @@
             % projection class.
             %
             if nargin == 1
+                obj = obj.check_combo_arg();
```

Comment:
any reason the check is not made at the end of the function as well? Even if it currently serves no purpose, having it there prevents forgetting it later

Reply:
standard serailizable operations over multiple properties do this check themselves enabling it in the correct place. 

from_bare_struct though does not so I've added the check to the counstructor